tests/spec/float_series_ops/test_division_ops.py

```diff
@@ -0,0 +1,127 @@
+from ..tables import p
+
+title = "Arithmetic division operations"
+
+table_data = {
+    p: """
+          |  f1   |   f2
+        --+-------+--------
+        1 | 101.3 |  111.5
+        2 |  -1.3 |  111.5
+        3 | -10.3 |    1.5
+        4 | -10.3 |   -1.5
+        5 |       |   -1.5
+        """,
+}
+
+
+def test_truedivide(spec_test):
+    spec_test(
+        table_data,
+        p.f1 / p.f2,
+        {
+            1: 101.3 / 111.5,
+            2: -1.3 / 111.5,
+            3: -10.3 / 1.5,
+            4: -10.3 / -1.5,
+            5: None,
+        },
+    )
+
+
+def test_truedivide_by_constant(spec_test):
+    spec_test(
+        table_data,
+        p.f1 / 10.0,
+        {
+            1: 101.3 / 10.0,
+            2: -1.3 / 10.0,
+            3: -10.3 / 10.0,
+            4: -10.3 / 10.0,
+            5: None,
+        },
+    )
+
+
+def test_rtruedivide_by_constant(spec_test):
+    spec_test(
+        table_data,
+        10.0 / p.f1,
+        {
+            1: 101.3 / 10.0,
+            2: -1.3 / 10.0,
+            3: -10.3 / 10.0,
+            4: -10.3 / 10.0,
+            5: None,
+        },
+    )
+
+
+def test_truedivide_by_zero(spec_test):
```

Comment:
I think that, rather than having a separate divide-by-zero test, it might be better to add another row to `table_data` where `f2` is zero and then that case gets covered in the other tests.

Reply:
yes, that makes sense. I remember getting stuck when I first tried this but found out why and changed it now